luma/core/device.py

```diff
@@ -168,6 +173,89 @@ def display(self, image):
         :param image: Image to display.
         :type image: PIL.Image.Image
         """
-        assert(image.size == self.size)
+        assert image.size == self.size
 
         self.image = self.preprocess(image).copy()
+
+
+def _toRGB565(image):
```

Comment:
these helpers belong in `luma.core.util`

Reply:
tempted not to do that - these implementations are specific to how the RGB pixels are structured in the framebuffer.

the 1-bit framebuffer conversion is a bit weird, and i actually decided to take it out along with the 32-bit ARGB version on the basis that we can add them at a later date if there is a demand (unlikely)